src/ActionModal/utils/ActionManager.js

```diff
@@ -4,12 +4,7 @@ import transaction from "./transactionTypes"
 import { uatoms } from "scripts/num"
 import { toMicroDenom } from "src/scripts/common"
 import { getGraphqlHost } from "scripts/url"
-import {
-  getMessage,
-  getMultiMessage,
-  getTransactionSigner,
-  transformMessage
-} from "./MessageConstructor.js"
+import { signedTransactionCreator, getMessage } from "./MessageConstructor.js"
```

Comment:
👀  OK, so it doesn't deprecate #3268 then. Here is our friend `signedTransactionCreator` again (whose naming I don't like 😛 )

Reply:
Why don't you like the name? I think the link you posted is wrong